Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockObjectFactory.cs

```diff
@@ -52,8 +53,8 @@ public MockObjectFactory(bool withDefaultRules = true, string organization = nul
             }
         }
 
-        public ISonarWebService CreateSonarWebService(ProcessedArgs args) =>
-            Server;
+        public Task<ISonarWebService> CreateSonarWebService(ProcessedArgs args, IDownloader downloader = null) =>
```

Comment:
The task is `async` now since it gets the version from the server. Will explain in the factory why this is needed.